addons/resource_mail/static/src/views/fields/many2many_avatar_resource/many2many_avatar_resource_field.xml

```diff
@@ -2,16 +2,34 @@
 <templates>
     <t t-name="resource_mail.AvatarResourceMany2XAutocomplete" t-inherit="web.AvatarMany2XAutocomplete">
         <xpath expr="//span[hasclass('o_avatar_many2x_autocomplete')]/img" position="before">
-            <i t-if="option.resourceType === 'material'" class="o_material_resource fa fa-wrench rounded text-center me-1"/>
+            <i t-if="option.resourceType === 'material'"
+               class="o_material_resource fa fa-wrench rounded text-center me-1"
+               t-attf-class="o_colorlist_item_color_{{ option.colorIndex }}"
+            />
         </xpath>
         <xpath expr="//span[hasclass('o_avatar_many2x_autocomplete')]/img" position="attributes">
             <attribute name="t-if" add="&amp;&amp; option.resourceType !== 'material'" separator=" "/>
         </xpath>
     </t>
 
     <t t-name="resource_mail.Many2ManyAvatarResourceTagsList" t-inherit="mail.Many2ManyAvatarUserTagsList">
-        <xpath expr="//span[hasclass('o_tag')]/i" position="attributes">
+        <i position="attributes">
             <attribute name="t-on-click.stop.prevent">tag.onImageClicked</attribute>
+            <attribute name="class">o_avatar o_m2m_avatar position-relative rounded</attribute>
+            <attribute name="t-attf-class" separator=" " add="o_colorlist_item_color_{{ tag.colorIndex }}"/>
+        </i>
+        <xpath expr="//span[hasclass('o_tag')]" position="attributes">
+            <attribute name="t-attf-class"/>
```

Comment:
What does this do? I'm curious :thinking:

Reply:
it unsets the attribute